### PR TITLE
Docs hpcc

### PR DESCRIPTION
**Pull request by jbrundage**

### CODE REVIEW

**Comment by jbrundage** [comment]
@GordonSmith

**Comment by GordonSmith** [comment]
I think its missing some CSS?

image

**Comment by jbrundage** [comment]
@GordonSmith this PR has been amended with the missing CSS
image

File: website/index.html [hunk 1]
```diff
@@ -32,22 +35,80 @@
...
+            --content-max-width: calc(100% - var(--leftnav-min-width) - var(--rightnav-min-width));
+            --content-background: #fff;
+
+            --topnav-font-size: 16px;
+            --topnav-item-horiz-padding: calc(var(--p) * 2);
+            --topnav-height: calc(var(--p) * 6);
+            --topnav-line-height: calc(var(--topnav-height) - (var(--p) * 2));
+            --topnav-border-bottom: 1px solid var(--border-color);
+            --topnav-background: #fff;
+        }
```
Comment (jbrundage) [inline website/index.html]: @GordonSmith the main aspects of the design can be modified here

**Pull request closed without merging**